server.js

```diff
@@ -71,8 +71,8 @@ var PATH = 'http://api.openweathermap.org/data/2.5/weather';
       // AND A ERROR OR SUCCESS FLAG
       // mainObj.cities will be a list of city objects name and ID keys 
       var mainObj = {};
-      len1Re = /[A-Za-z\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF]/;
-      trimmed = clientReq['q'].split(',').map(function (element) {
+      const len1Re = /[A-Za-z\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF]/;
```

Comment:
this variable (and trimmed below) were declared without a keyword so they are globals. probably don't want that. 

made the regex const -- it shouldn't change. trimmed might be able to be const as well, but that depends on how you use it.